### PR TITLE
Add logging for Connector Service

### PR DESCRIPTION
**Pull request by Maladie**
**Description**

Changes proposed in this pull request:

- Adds logging with Application, Group and Tennant fields to Connector-service

**Related issue(s)**

#2613

### CODE REVIEW

File: components/connector-service/internal/clientcontext/model.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ type ConnectorClientContext interface {
 	GetApplication() string
 	ToJSON() ([]byte, error)
 	GetCommonName() string
+	GetTenant() string
+	GetGroup() string
```
[the remark this answers is not in the document]
Comment (franpog859) [reply components/connector-service/internal/clientcontext/model.go]: Done!

File: components/connector-service/internal/externalapi/signaturehandler.go [hunk 2]
```diff
@@ -38,20 +39,28 @@ func (sh *signatureHandler) SignCSR(w http.ResponseWriter, r *http.Request) {
...
+
+	logger.Info("Reading certificate signing request")
 	signingRequest, err := sh.readCertRequest(r)
 	if err != nil {
+		logger.Error(err)
 		httphelpers.RespondWithError(w, err)
 		return
 	}
 
+	logger.Info("Decoding certificate signing request from base64")
```
[the remark this answers is not in the document]
Comment (franpog859) [reply components/connector-service/internal/externalapi/signaturehandler.go]: Done!

File: components/connector-service/internal/logging/contextlogger.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package logging
+
+import (
+	"github.com/kyma-project/kyma/components/connector-service/internal/clientcontext"
+	log "github.com/sirupsen/logrus"
+)
+
+func GetLogger(ctx clientcontext.ConnectorClientContext) *log.Entry {
```
[the remark this answers is not in the document]
Comment (franpog859) [reply components/connector-service/internal/logging/contextlogger.go]: Done!

File: components/connector-service/internal/logging/contextlogger.go [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+)
+
+func GetClusterLogger(tenant, group string) *log.Entry {
+	return log.WithFields(log.Fields{
+		"Group":  group,
+		"Tenant": tenant,
+	})
+}
+
+func GetApplicationLogger(application, tenant, group string) *log.Entry {
```
Comment (Szymongib) [inline components/connector-service/internal/logging/contextlogger.go]: We could simplify this method to something like that:
```go
func GetApplicationLogger(application, tenant, group string) *log.Entry {
	return GetClusterLogger(tenant, group).WithField("application", application)
}
```

Comment (franpog859) [reply components/connector-service/internal/logging/contextlogger.go]: You are right! Done ;>

**Pull request merged**